tests/keras_contrib/callbacks/warm_restart_test.py

```diff
@@ -0,0 +1,42 @@
+from keras.models import Sequential
+from keras.layers.core import Dense
+from keras.utils.test_utils import get_test_data
+from keras import backend as K
+from keras.utils import np_utils
+from keras_contrib import callbacks
+
+import numpy as np
+import pytest
+
+input_dim = 2
+num_hidden = 4
+num_classes = 2
+batch_size = 5
+train_samples = 20
+test_samples = 20
+
+
+def LearningRateWarmRestarter():
```

Comment:
could this be named something other than the class name in warm_restart.py?

Reply:
done. Thanks!
`LearningRateWarmRestarter` -> `test_LearningRateWarmRestarter`